### PR TITLE
fix bug on chat input

### PR DESCRIPTION
**Pull request by narender2031**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
- In connect input. When user press `shift`+ `enter`. it was clearing the text input. 

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by narender2031** [comment]
@nickytonline Please restart the build

**Comment by narender2031** [comment]
any updates on this @citizen428 ?

**Review by nickytonline — CHANGES_REQUESTED** [review]
tests (blocking): Add some tests for this change.

**Comment by narender2031** [comment]
> tests (blocking): Add some tests for this change.
@nickytonline 
done

File: app/javascript/chat/__tests__/compose.test.jsx [hunk 1]
```diff
@@ -87,6 +90,16 @@ describe('<Compose />', () => {
       expect(submitWithMessage).toEqual(false);
       expect(textfieldIsEmpty).toEqual(true);
     });
+
+    it('should pressed enter and shift', () => {
```
Comment (citizen428) [inline app/javascript/chat/__tests__/compose.test.jsx:94]: **nitpick (non-blocking):** I think this message could be worded a bit differently, but we can do that later.

**Review by citizen428 — APPROVED** [review]

**Review by nickytonline — APPROVED** [review]

**Comment by nickytonline** [comment]
Thanks for the PR @narender2031 and thanks for updating the tests.

**Pull request merged**